mod/widget_manager/lib/widgets.php

```diff
@@ -68,7 +68,7 @@ function widget_manager_widgets_init() {
 
 	// messages
 	if (elgg_is_active_plugin("messages")) {
-		elgg_register_widget_type("messages", elgg_echo("messages"), elgg_echo("widgets:messages:description"), array("dashboard", "index"), false);
+		elgg_register_widget_type("messages", elgg_echo("Notifications"), elgg_echo("widgets:messages:description"), array("dashboard", "index"), false);
```

Comment:
Not a big deal (since it only displays for you) but maybe use `elgg_echo('messages:notifications');` instead of Notifications, for best practices.